src/main/java/TaskManager.java

```diff
@@ -0,0 +1,61 @@
+public class TaskManager {
```

Comment:
I like how you separated the task functions in to a TaskManager class. Code is structured logically.